components/brave_ads/core/internal/account/account_util_unittest.cc

```diff
@@ -0,0 +1,130 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// npm run test -- brave_unit_tests --filter=BraveAds*
+
+#include "brave/components/brave_ads/core/internal/account/account_util.h"
+
+#include "base/test/scoped_feature_list.h"
+#include "brave/components/brave_ads/core/internal/common/unittest/unittest_base.h"
+#include "brave/components/brave_ads/core/internal/settings/settings_unittest_util.h"
+#include "brave/components/brave_ads/core/public/account/confirmations/confirmation_type.h"
+#include "brave/components/brave_ads/core/public/ads_feature.h"
+
+namespace brave_ads {
+
+class BraveAdsAccountUtilTest : public UnitTestBase {};
+
+TEST_F(BraveAdsAccountUtilTest, AlwaysAllowDepositsForRewardsUser) {
+  // Act & Assert
+  for (auto i = 0; i < static_cast<int>(AdType::kMaxValue); ++i) {
```

Comment:
maybe `for (int i = 0;` as it is a more widely used pattern in chromium.

Reply:
Fixed. Great catch.

Reply:
Fixed. Thanks

Reply:
Fixed